datacite.yml

```
@@ -0,0 +1,97 @@
+# Metadata for DOI registration according to DataCite Metadata Schema 4.1.
+# For detailed schema description see https://doi.org/10.5438/0014
+
+authors:
+  - firstname: Michael
+    lastname: Hanke
+    id: "ORCID:0000-0001-6398-6370"
+
+  - firstname: Falko R.
+    lastname: Kaule
+
+  - firstname: Ayan
+    lastname: Sengupta
+
+  - firstname: J. Swaroop
+    lastname: Guntupalli
+
+  - firstname: Christian
+    lastname: Häusler
+    id: "ORCID:0000-0002-0936-317X"
+
+  - firstname: Michael
+    lastname: Hoffmann
+
+  - firstname: Jörg
+    lastname: Stadler
+
+  - firstname: Adina
+    lastname: Wagner
+    id: "ORCID:0000-0003-2917-3450"
+
```

```suggestion

  - firstname: Laura
    lastname: Waite
    id: "ORCID:0000-0003-2213-7465"
```
I think we should be consistent and either include all original authors only, or original authors plus git contributors (and I think including Git contributors is good). In the latter case, you and laura are missing from those author lists. :)